src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -30,9 +59,14 @@ class InputView {
         return readLine()?.toIntOrNull() ?: 0
     }
 
-    private fun isValid(number: Int): Boolean {
-        if (number in PURCHASABLE_RANGE) return true
-        return false
+    private fun inputLottoNumbers(): List<Int> {
+        val input = readLine()?.trim()
+        val numbers = input?.split(",", " ")?.map { it.trim().toInt() }
+        return numbers?.take(Lotto.COUNT_OF_LOTTO_NUMBER) ?: emptyList()
```

Comment:
null safe call 연산(`?.`)을 체이닝하면서 사용하면 가독성을 해칠 수도 있을 것 같아요.
필요한 경우 최상단에서 null을 제거해주면 어떨까요?
```suggestion
        val input = readlnOrNull() ?: return emptyList()
        val numbers = input.split(",", " ")
            .map { it.trim().toInt() }
        return numbers.take(Lotto.COUNT_OF_LOTTO_NUMBER)
```